autoload/lsp/utils.vim

```diff
@@ -68,10 +68,16 @@ endif
 
 if has('win32') || has('win64')
     function! lsp#utils#uri_to_path(uri) abort
```

Comment:
any changes to path requires tests as it is very error prone and is the core of vim-lsp as it can break anything.